scripts/clean_osm_data.py

```diff
@@ -342,6 +342,7 @@ def clean_frequency(df, default_frequency="50"):
         "16.67": "16.7",
         "50;50;16.716.7": "50;50;16.7;16.7",
         "50;16.7?": "50;16.7",
+        "50.0": "50",
         # "24 kHz": "24000",
```

Comment:
May it make sense to add also 60.0?
```suggestion
        "50.0": "50",
        "60.0": "60",
        # "24 kHz": "24000",
```